### DISCUSSION TITLE
Video

### DISCUSSION THREAD

**Discussion by VlaK0r** [Athou/commafeed#1079] [Q&A]
Hello!
Does it turn out that the video cannot be viewed in the feed?
Or am I doing something wrong?

**Comment by Athou** [comment]
It all depends on the feed, if a link to the video is supplied it will be shown.

**Reply by VlaK0r** [reply]
For example, a video from YouTube?

**Reply by VlaK0r** [reply]
For example, here is this link. Should show video
http://195.133.13.45:3000/?action=display&bridge=TelegramBridge&username=%40prozhara&format=Mrss
